### PR TITLE
Make `node-id` required for quickstart-pytorch

### PR DESCRIPTION
**Pull request by neogismm**
### Related issues/PRs
Related to issue #2711

### Checklist

- [ ] Implement proposed change
- [ ] Write tests
- [ ] Update [documentation](https://flower.dev/docs/writing-documentation.html)
- [ ] Update [changelog](https://github.com/adap/flower/blob/main/doc/source/changelog.rst)
- [ ] Make CI checks pass
- [ ] Ping maintainers on [Slack](https://flower.dev/join-slack/) (channel `#contributions`)

### CODE REVIEW

File: examples/quickstart-pytorch/client.py [hunk 1]
```diff
@@ -97,7 +97,7 @@ def apply_transforms(batch):
 # Get node id
 parser = argparse.ArgumentParser(description="Flower")
 parser.add_argument(
-    "--node-id",
+    "node-id",
```
Comment (jafermarq) [inline examples/quickstart-pytorch/client.py:100]:
```suggestion
    "--node-id",
```

File: examples/quickstart-pytorch/run.sh [hunk 2]
```diff
@@ -8,7 +8,7 @@ sleep 3  # Sleep for 3s to give the server enough time to start
 
 for i in $(seq 0 1); do
     echo "Starting client $i"
-    python client.py --node-id "$i" &
+    python client.py node-id "$i" &
```
Comment (jafermarq) [inline examples/quickstart-pytorch/run.sh:11]:
```suggestion
    python client.py --node-id "$i" &
```

File: examples/quickstart-pytorch/client.py [hunk 3]
```diff
@@ -97,7 +97,7 @@ def apply_transforms(batch):
...
     choices=[0, 1, 2],
     type=int,
```
Comment (jafermarq) [inline examples/quickstart-pytorch/client.py:102]:
```suggestion
    type=int,
    required=True,
```

Comment (jafermarq) [reply examples/quickstart-pytorch/client.py:102]: This is what i meant when i commented on your issue. With this, if someone runs the code without passing `--node-id` an error will be thrown.

**Review by jafermarq — COMMENTED** [review]
Hi @neogismm,

Thanks for opening the PR. I added a small comment. I think the best is to edit the `argparse` and make the `node-id` to be "required"

**Review by jafermarq — APPROVED** [review]
@neogismm ,

Many thanks for the PR! 🙌

**Comment by neogismm** [comment]
> Hi @neogismm,
> 
> Thanks for opening the PR. I added a small comment. I think the best is to edit the `argparse` and make the `node-id` to be "required"

I found that removing "--" before an arg makes it a required arg. Nonetheless, your solution does the same thing.
Thanks for the opportunity @jafermarq 🙌

**Pull request merged**